tests/backup/backup_restore_basic_test.go

```diff
@@ -658,7 +666,7 @@ var _ = Describe("{ScheduleBackupCreationAllNS}", func() {
 
 	JustBeforeEach(func() {
 		StartTorpedoTest("ScheduleBackupCreationAllNS", "Create schedule backup creation with all namespaces", nil, testrailID)
-		log.Info("Application installation")
+		log.Infof("Application installation")
 		contexts = make([]*scheduler.Context, 0)
 		for i := 0; i < Inst().GlobalScaleFactor; i++ {
```

Comment:
You can address this in a separate PR. If only one application namespace is created, there would not be any difference between ScheduleBackupCreationSingleNS and ScheduleBackupCreationAllNS. 

For ScheduleBackupCreationAllNS to have relevance in such scenario, you can replace the loop 

`for i := 0; i < Inst().GlobalScaleFactor; i++` with 

`for i := 0; i < numDeployments; i++`, where numDeployments is set to 2 or a value greater than 1. @vpinisetti-px Please advice on this.

Reply:
Ok will address this in another PR. Looks like many testcases need to be modified.